### ISSUE TITLE
latest graphql causes error again!

### ISSUE THREAD

**Issue by sangche** [Canner/gqlify#55]
**Describe the bug**
when try [quickstart](https://www.gqlify.com/docs/quick-start) in codesandbox,
the latest dependency graphql 14.2.1 throws the following error:
```
/sandbox/node_modules/@gqlify/server/lib/rootNode.js:247
        this.defBuilder._typeDefinitionsMap[name] = typeDefNode;
                                                  ^

TypeError: Cannot set property 'PageInfo' of undefined
    at RootNode.buildObjectType (/sandbox/node_modules/@gqlify/server/lib/rootNode.js:247:51)
    at RootNode.addObjectType (/sandbox/node_modules/@gqlify/server/lib/rootNode.js:93:23)
    at RelayPlugin.init (/sandbox/node_modules/@gqlify/server/lib/plugins/relay.js:58:14)
    at /sandbox/node_modules/@gqlify/server/lib/generator.js:22:24
    at Array.forEach (<anonymous>)
    at Generator.generate (/sandbox/node_modules/@gqlify/server/lib/generator.js:17:22)
    at Gqlify.createServerConfig (/sandbox/node_modules/@gqlify/server/lib/gqlify.js:107:34)
    at Gqlify.createApolloConfig (/sandbox/node_modules/@gqlify/server/lib/gqlify.js:116:33)
    at Object.<anonymous> (/sandbox/index.js:28:40)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
[nodemon] app crashed - waiting for file changes before starting...
```
**To Reproduce**
run on [codesadgbox](https://codesandbox.io/s/kkn3km12jo)

**Expected behavior**
run as normal

**Screenshots**

**Desktop (please complete the following information):**
 - Browser chrome

**Additional context**
explicit downgrade to graphql 14.1.1 fixed the problem

**Comment by alexandrethsilva** [comment]
Duplicate of #54.
